gpkitmodels/GP/aircraft/wing/gustloading.py

```diff
@@ -49,7 +51,7 @@ class GustL(SparLoading):
     new_SbarFun = None
 
     return_cosm1 = lambda self, c: hstack(
-        [1e-10, 1-cos(c[self.wing.planform.eta][1:]*pi/2)])
+        [adnumber(1e-10), 1-array(cos(c[self.wing.planform.eta][1:]*pi/2))])
```

Comment:
the array is necessary because admath.cos returns a list, unfortunately.

Note that if you wrapped the whole thing in adnumber you'd wipe out all derivative information!